### PR TITLE
Add,edit,show,delete Employee

### PR DESCRIPTION
**Pull request by ChinIndival**

### CODE REVIEW

**Comment by ChinIndival** [comment]
ready

**Comment by ChinIndival** [comment]
Ready

**Comment by ChinIndival** [comment]
Ready

**Comment by ChinIndival** [comment]
Ready

File: app/Http/Controllers/Manager/UsersController.php [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+     * Display the specified resource.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $user= User::findOrFail($id);
+
+        return view('backend.users.show',compact('user'));
```
Comment (BunnyPi04) [inline app/Http/Controllers/Manager/UsersController.php]: convention

**Comment by ChinIndival** [comment]
ready

File: app/Http/Controllers/Manager/UsersController.php [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        $user->email = $request->get('email');
+        $user->role = $request->get('role');
+        $password = $request->get('password');
+            if($password != "") {
+                $user->password = Hash::make($password);
+            }
+        $user->phone = $request->get('phone');
+        $user->sex = $request->get('sex');
+        $user->address = $request->get('address');
+        $user->save();
```
Comment (BunnyPi04) [inline app/Http/Controllers/Manager/UsersController.php]: sử dụng scope hoặc repository nhé

File: app/Http/Controllers/Manager/UsersController.php [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+        return redirect(action('Manager\UsersController@index', $user->id))->with('status', __('updated'));
```
Comment (BunnyPi04) [inline app/Http/Controllers/Manager/UsersController.php]: redirect về route nhé

**Comment by BunnyPi04** [comment]
commented

**Comment by ChinIndival** [comment]
Ready

**Comment by ChinIndival** [comment]
ready

**Comment by ChinIndival** [comment]
ready

File: app/Http/Controllers/Manager/UsersController.php [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        $this->userRepository = $userRepository;
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+	{
+		$users = $this->userRepository->getUser();
```
Comment (BunnyPi04) [inline app/Http/Controllers/Manager/UsersController.php]: tab lại

File: resources/views/backend/users/create.blade.php [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+                                <div class="col-lg-12 col-lg-offset-6">
+                                    {!! Form::open(['method' => 'POST', 'url' => asset('manager/users')]) !!}
+                                        <div class="form-group">
+                                            {{ Form::label(__('name'), null, ['class' => 'control-label']) }}
+                                            {{ form::text('name', null, ['class' => 'form-control']) }}
+                                        </div>
+                                          
+                                         <div class="form-group">
+                                            {{ Form::label(__('email'), null, ['class' => 'control-label']) }}
+                                            {{Form::text('email', '@gmail.com', ['class' => 'form-control']) }}
```
Comment (BunnyPi04) [inline resources/views/backend/users/create.blade.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by ChinIndival** [comment]
ready

**Pull request merged**